include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -260,4 +262,91 @@ def YieldOp : VerifOp<"yield", [
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// Verification intent ops
+//===----------------------------------------------------------------------===//
+
+/// BMC -- short for bounded model check.
+def BMC : I32EnumAttrCase<"BMC", 0, "bmc">;
+/// KIP -- short for k-Inductive-Proof.
```

Comment:
Strange comment, because you're not using this acronym anymore.

Reply:
lol true